source/renderer/app/containers/wallet/dialogs/settings/UndelegateWalletDialogContainer.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import { get, find } from 'lodash';
+import type { StoresMap } from '../../../../stores/index';
+import type { ActionsMap } from '../../../../actions/index';
+import type { DelegationCalculateFeeResponse } from '../../../../api/staking/types';
+import UndelegateWalletConfirmationDialog from '../../../../components/wallet/settings/UndelegateWalletConfirmationDialog';
+import UndelegateWalletConfirmationResultDialog from '../../../../components/wallet/settings/UndelegateWalletConfirmationResultDialog';
+import { DELEGATION_ACTIONS } from '../../../../config/stakingConfig';
+
+type Props = {
+  stores: any | StoresMap,
+  actions: any | ActionsMap,
```

Comment:
@yakovkaravelov I'm not sure that this `any` declaration is ok. We have store types declared e.g. `InjectedDialogContainerProps` and similar.

Reply:
👍